vmas/simulator/environment/environment.py

```diff
@@ -468,6 +468,7 @@ def render(
         mode="human",
         env_index=0,
         agent_index_focus: int = None,
+        scene_focus: bool = False,
```

Comment:
```suggestion
        scene_focus: bool = True,
```

This works! Can we set it to default True or is there a reason you wanted False?

Reply:
nope, I'll set it to true